### PR TITLE
[Improve](schemaChange)schema change support rename column

### PR DESCRIPTION
**Pull request by DongLiang-0**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:
schema change support rename column.
## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/SchemaChangeHelper.java [hunk 1]
```diff
@@ -57,6 +58,24 @@ public static void compareSchema(Map<String, FieldSchema> updateFiledSchemaMap,
...
+                String renameSQL = String.format(RENAME_DDL, table, oldColumnName, newColumnName);
+                ddlList.add(renameSQL);
+                ddlSchemas.add(new DDLSchema(oldColumnName, false));
+            }
+        }
+        originFieldSchemaMap.remove(oldColumnName);
+        originFieldSchemaMap.put(newColumnName, fieldSchema);
+        return ddlList;
+    }
+
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/SchemaChangeHelper.java:78]: please add ut，thanks

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/JsonDebeziumSchemaSerializer.java [hunk 2]
```diff
@@ -249,12 +251,24 @@ public List<String> extractDDLList(JsonNode record) throws JsonProcessingExcepti
...
+
+        // rename ddl
+        Matcher renameMatcher = renameDDLPattern.matcher(ddl);
+        if (renameMatcher.find()) {
+            String oldColumnName = renameMatcher.group(2);
+            String newColumnName = renameMatcher.group(3);
+            return SchemaChangeHelper.generateRenameDDLSql(
+                    dorisOptions.getTableIdentifier(), oldColumnName, newColumnName, originFieldSchemaMap);
+        }
+
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/JsonDebeziumSchemaSerializer.java:263]: please add ut，thanks

Comment (DongLiang-0) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/JsonDebeziumSchemaSerializer.java:263]: done!

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/SchemaChangeHelper.java (hunk 1 above)
Comment (DongLiang-0) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/SchemaChangeHelper.java:78]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**